packages/htmlbars-compiler/lib/utils.js

```diff
@@ -9,3 +9,15 @@ export function processOpcodes(compiler, opcodes) {
     }
   }
 }
+
+export function getNamespace(name) {
+	var rval = {ns: null, name: null};
```

Comment:
Please use standard two-space indentation, and avoid abbreviations (`ns`). And a map:

```
// Map values should be based on http://dev.w3.org/html5/spec-LC/namespaces.html
var defaultNamespaces = {
  xlink: {
    namespaceURI: 'http://www.w3.org/1999/xlink',
    name: 'xlink'
  }
};

export function getNamespace(attrName) {
  var parts = attrName.split(':');
  if (parts.length > 1) {
    return defaultNamespaces[parts[0]];
  }
}
```